#request
uint8 start_stop
uint8 payload_index
---
#response
bool result